src/test/java/com/wikia/webdriver/common/core/imageutilities/SzogiScreenshotTests.java

```diff
@@ -0,0 +1,69 @@
+package com.wikia.webdriver.common.core.imageutilities;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import org.testng.annotations.Test;
+
+
+    @Test(groups = "SzogiScreenshotTests")
```

Comment:
can you do it on some regular wiki? Like mediawiki119?

Reply:
Sure, I just copied it from some other tests